lib/VerifiableCredentialStore.js

```diff
@@ -113,6 +113,42 @@ export default class VerifiableCredentialStore {
     return doc.content;
   }
 
+  /**
+   * Updates a verifiable credential in remote private storage
+   *
+   * @param {object} credential
+   * @param {object} meta
+   */
+  async update({credential, meta = {}}) {
+    try {
+      const {invocationSigner} = this;
+      const {documents: [doc]} = await this.edv.find({
+        equals: {'content.id': credential.id},
+        invocationSigner
+      });
```

Comment:
Hmm, I'm not sure if this is going to be safe for the use cases we have. We don't want to lose any concurrent updates that came in via another device, right? If device 1 updated `meta` while we were doing something and now we want to update `meta` with device 2, we want to make sure we don't overwrite/lose the changes from device 1, right? That implies we want to pass along the sequence number or full EDV doc here (or use the EDV doc instance update interface, if that's a thing). And then, if there's a conflict, make the appropriate adjustment.

Can you explore what can be done to accommodate this?